### PR TITLE
adding two new domains, federator (optional) and sftd

### PR DESCRIPTION
**Pull request by arthurwolf**
## Checklist:

Please tick the following before making your PR:

* [x] I ran `make` on this branch to build the docs and there are **no WARNINGs**.
* [x] After running `make`, I looked at the output by opening build/html/index.html in a browser and I'm satisfied with the rendering.

### CODE REVIEW

File: src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-An installation needs 5 or 6 domain names (5 without audio/video support, 6 with audio/video support):
+An installation needs 6 or 7 domain names (6 without audio/video support, 7 with audio/video support):
 
 You need
 
 * two DNS names for the so-called "nginz" component of wire-server (the main REST API entry point), these are usually called `nginz-https.<domain>` and `nginz-ssl.<domain>`.
 * one DNS name for the asset store (images, audio files etc. that your users are sharing); usually `assets.<domain>` or `s3.<domain>`.
 * one DNS name for the webapp (equivalent of https://app.wire.com, i.e. the javascript app running in the browser), usually called `webapp.<domain>`.
 * one DNS name for the account pages (hosts some html/javascript pages for e.g. password reset), usually called `account.<domain>`.
+* one DNS name for SFTD support (conference calling), usually called `sftd.<domain>`
```
Comment (julialongtin) [inline src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst:12]:
```suggestion
* (optional) one DNS name for SFTD support (conference calling), usually called `sftd.<domain>`
```

File: src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst [hunk 2]
```diff
@@ -20,14 +22,15 @@ If you are on the most recent charts from wire-server-deploy, these are your nam
 * assets.<domain>
 * account.<domain>
 * teams.<domain>
+* sftd.<domain>
```
Comment (julialongtin) [inline src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst:25]: sft is 'just as optional' as restund.

File: src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst [hunk 3]
```diff
@@ -19,15 +21,31 @@ If you are on the most recent charts from wire-server-deploy, these are your nam
...
+* federator.<domain>
+
+All of these DNS records need to point to the same IP address, the IP you want to provide services on.
+
+This is necessary for the nginx ingress to know how to do internal routing based on virtual hosting.
+
+The only expections to this are:
+
+* restund01, restund02  which need the appropriate DNS name pointed to them
+* sftd which needs to point to both of the external IPs you are providing conference calling on
```
Comment (julialongtin) [inline src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst:40]: this is only true if you're using a separate k8s cluster to run SFT on.. which so far only our largest customers have wanted.

Comment (arthurwolf) [reply src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst:40]: What do you recommend I say here then? Adding at the end «, only if SFT is run on a separate K8S cluster» ?
Maybe there's something I could link to to explain the difference/situation? Or there's some internal document I can port information from?

Comment (julialongtin) [reply src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst:40]: hmm. i think omiting it is fine. not that we only have one external IP we provide services on under k8s, so 'both of' on line 40 can be dropped.

Comment (arthurwolf) [reply src/how-to/install/includes/helm_dns-ingress-troubleshooting.inc.rst]: removed "both of" in latest commit

**Pull request merged**